### ISSUE TITLE
Add account created modal to end of onboarding upon app auth

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/stacks-wallet-web#1808]
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/📏Spec?node-id=80%3A2688

![image](https://user-images.githubusercontent.com/28991/136820793-5a68e536-b8ef-4aad-9e33-8b0bd77cf416.png)

@eugeniadigon to replace the checkmark graphic with an illustration

**Comment by ginny-d** [comment]
Updated icon: 

https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=80%3A2688

<img width="1352" alt="Home" src="https://user-images.githubusercontent.com/78865845/150517264-0971b095-c339-4b55-ac6d-461bd0c674b5.png">

**Comment by fbwoolf** [comment]
What do we want to do here if the `redirectTo` prop is not provided by the app? It is optional in Connect.

**Comment by markmhendrickson** [comment]
I suspect we should just not show this modal if there’s no `redirectTo` property. cc @landitus

**Comment by markmhx** [comment]
I suspect we should just not show this modal if there’s no `redirectTo` property. cc @landitus

**Comment by landitus** [comment]
When is this modal triggered today? Can you help me reproduce it?

**Comment by landitus** [comment]
I see, with this modal we would provide a path back to the original app after account creation. I think this piece will be helpful to return the use to original context. Is this the intent of this step?

Are we adding tracking to this view and to the interactive elements?

**Comment by fbwoolf** [comment]
Yep, @landitus, that is the intent. So, if a user onboards from an app it will send them back to it. We can def add tracking to it. 👍

**Comment by fbwoolf** [comment]
Going to start this and add it to the home page branch.

**Comment by fbwoolf** [comment]
I implemented this but it doesn't totally make sense with our current onboarding flow ...not really sure how a user even arrives here after downloading/installling and onboarding via full page. We can discuss after retreat.

**Comment by fbwoolf** [comment]
Going to drop an old PR link here bc I started some work on this a long time ago when I first joined the UX team. If we really want to route a user back to the app after onboarding we will have to rework Connect bc the user will be coming from the Chrome Store (or FF) after download/install not from the app at that point. https://github.com/hirosystems/stacks-wallet-web/pull/1307 (going to close this draft PR bc it is really outdated).

**Comment by andresgalante** [comment]
@fbwoolf to spend 30mins looking at the solution, if this issue is too complicated, let's move it back to the backlog

**Comment by fbwoolf** [comment]
I think we should move this into the backlog and revisit it. I think there are security implications with passing the app url thru the query params for redirecting. This was my original work on this in Connect and here for reference:

https://github.com/hirosystems/connect/pull/148
https://github.com/hirosystems/stacks-wallet-web/pull/1307

Jasper had originally wanted to address this thru Connect and offering an onboarding library similar to MetaMask:
https://docs.metamask.io/guide/onboarding-library.html#onboarding-library

I think this is more inline with what @landitus has suggested doing, so it makes sense to hold on this until he can make a pass at how we refactor onboarding again that better integrates with Connect (download/install modal).

**Comment by landitus** [comment]
Thanks @fbwoolf! I'll look into the Metamask reference in the meantime.
